src/TransactionBuilder.php

```diff
@@ -156,30 +164,48 @@ public function updateOutputValue($idx, $value) {
      * $data will be bin2hex and will be prefixed with a proper OP_PUSHDATA
      *
      * @param string $data
-     * @param bool   $allowNonStandard when TRUE will allow scriptPubKey > 40 bytes (so $data > 39 bytes)
+     * @param bool   $allowNonStandard  when TRUE will allow scriptPubKey > 80 bytes (so $data > 80 bytes)
      * @return $this
      * @throws BlocktrailSDKException
      */
     public function addOpReturn($data, $allowNonStandard = false) {
-        $pushdata = RawTransaction::pushdata(bin2hex($data));
-
-        if (!$allowNonStandard && strlen($pushdata) / 2 > 40) {
-            throw new BlocktrailSDKException("OP_RETURN data should be <= 39 bytes to remain standard!");
+        if (!$allowNonStandard && strlen($data) / 2 > 79) {
+            throw new BlocktrailSDKException("OP_RETURN data should be <= 79 bytes to remain standard!");
         }
 
+        $script = ScriptFactory::create()
+            ->op('OP_RETURN')
+            ->push(new Buffer($data))
```

Comment:
The documentation mentions data is converted from hex to binary, so this should be `Buffer::hex`